mne/stats/regression.py

```diff
@@ -134,3 +142,197 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
```

Comment:
Agreed that the coefficients are the things we interpret (though predictive
power is still an interesting thing to look at as well). However, I guess
my approach to regression is that I want to know a model is doing a "good"
(aka decent, aka probably-horrible-but-not-so-bad-for-brain-signals) before
I try to interpret it, which is why I like doing cross validations and
using generalization scores as a proxy for models that are worth looking
into more.

E.g., if you fit an auditory spectrogram model on a ton of ecog electrodes,
you'll get most scores between -.05 to .05, and a few that clearly pop out
in their predictive power, which helps me to have confidence that there's
something worth saying about those coefficients in general.

On Fri, Jul 17, 2015 at 11:55 AM, jona-sassenhagen <notifications@github.com
> wrote:

> In mne/stats/regression.py
> <https://github.com/mne-tools/mne-python/pull/2304#discussion_r34876664>:
>
> > @@ -134,3 +142,197 @@ def _fit_lm(data, design_matrix, names):
> >          mlog10_p_val[predictor] = -np.log10(p_val[predictor])
> >
> >      return beta, stderr, t_val, p_val, mlog10_p_val
> > +
>
> Note this part wasn't written by me, it's old - for continuous regression,
> we only output betas.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2304/files#r34876664>.
>